lib/src/base/_stacks-body.less

```diff
@@ -25,12 +25,12 @@ body {
     line-height: @lh-base;
 }
 
-@media (max-width: 640px) {
+#stacks-internals #screen-sm({
```

Comment:
It doesn't work if, like on the Q&A sites, responsiveness is controlled via CSS classes on the root element, because you get a selector like this:

```css
html.html__responsive html {
   ...
}
```